.github/workflows/tests.yml

```diff
@@ -17,6 +17,12 @@ permissions:
   pull-requests: write
 
 jobs:
+  BlockInfo:
```

Comment:
DOn't add these the normal Tests.
It's gonna end up being too large.
Please create a new workflow that can be kicked off manually for Solidity EVM equivalence coverage.
Can do this is a separate PR